src/main/xerus/monstercat/tabs/TabSettings.kt

```diff
@@ -80,6 +83,20 @@ class TabSettings: VTab() {
 			Settings.PLAYERSEEKBARHEIGHT.bind(valueProperty() as ObservableValue<out Double>)
 		})
 		
+		val coverPriority = ComboBox(FXCollections.observableArrayList<String>())
+		onFx {
+			PriorityLists.values().forEach { priorities ->
+				coverPriority.items.add(PriorityLists.getString(priorities))
+				coverPriority.valueProperty().bindBidirectional(Settings.PLAYERARTPRIORITY, {
```

Comment:
I'm pretty sure I have a util function for this whole piece of code, please check out other ComboBoxes in the project

Reply:
2045bffad1e8fcbc3a72fe235f504ce706ac903f